src/main/resources/react/src/pages/ErrorPages/styles.js

```diff
@@ -0,0 +1,186 @@
+import styled, { css } from 'styled-components';
+import SomethingBad from 'assets/img/errorPage/SomethingBad.png';
+
+export const WrapCenter = styled.div`
+  position: relative;
+  width: 596px;
+  margin: 278px auto 73px;
+`;
+
+export const WrapImg404 = styled(WrapCenter)`
+  position: relative;
+  z-index: 10;
+`;
+
+export const WrapperShadow = styled(WrapCenter)`
+  position: absolute;
+  left: calc(50% - (483px / 2));
+  bottom: -72px;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  width: 483px;
+  background-color: #e2e2e2;
+  z-index: 0;
+
+  img {
+    display: inline-flex;
+  }
```

Comment:
You can to create new style component

export const Image  = styled.img`
display: inline-flex;
`